.circleci/config.yml

```diff
@@ -475,6 +586,22 @@ workflows:
           test_tags: "bench"
           requires:
             - "packet-test-recover"
+      - example-test:
+          name: "packet-test-example-icmp"
+          namespace: "nsm-system"
+          cloud_provider: "packet"
+          cluster_id: "2"
+          example: "icmp"
+          requires:
+            - "packet-test-bench"
+      - example-test:
+          name: "packet-test-example-vpn"
+          namespace: "nsm-system"
+          cloud_provider: "packet"
+          cluster_id: "2"
+          example: "vpn"
+          requires:
+            - "packet-test-example-icmp"
```

Comment:
I'm not a circleci ninja, so pardon my question if it is dumb, but why you require packet-test-example-icmp here?

Reply:
So I can re-use the same cluster and queue both tests one after the other.